### PR TITLE
Clarify install steps

### PR DESCRIPTION
**Pull request by rowan-m**
- Clarify gulp installation
- Use full names for npm commands
- Move gulp dev step up to help verify a correct setup
- Clarify dev server location and opening

### CODE REVIEW

**Comment by rowan-m** [comment]
Fixed git author to be me, CLA bot should be happy now.

File: README.md [hunk 1]
```diff
@@ -11,16 +11,30 @@ In case we are missing any examples, feel free to [let us know](https://github.c
 ## Installation
 
 1. Fork the repository.
-2. Install [NodeJS](https://nodejs.org).
-3. Setup the repository.
+2. Install [NodeJS](https://nodejs.org). You will need version 4.0.0 or above.
+3. Install [Gulp](http://gulpjs.com/) via `npm`:
+   * *Note:* You may need to use `sudo` on the last command depending on your Node installation.
+
+```none
```
Comment (sebastianbenz) [inline README.md]: add indentation to the code sections to not break the list numbering.

File: README.md [hunk 2]
```diff
@@ -15,23 +15,23 @@ In case we are missing any examples, feel free to [let us know](https://github.c
 3. Install [Gulp](http://gulpjs.com/) via `npm`:
    * *Note:* You may need to use `sudo` on the last command depending on your Node installation.
```
Comment (sebastianbenz) [inline README.md]: nits: 

* I'd prefer to remove the bullet before Note (not a big fan of single bullet lists)
* remove "on the last command" as there is only one command left

**Comment by sebastianbenz** [comment]
Missed a few nits in my first review. LGTM once these are fixed. Thanks a lot!

**Comment by sebastianbenz** [comment]
@rowan-m can you please take another look.

**Comment by sebastianbenz** [comment]
LGTM Cheers!

**Pull request merged**